app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -756,61 +810,6 @@ class PasswordStore : AppCompatActivity(R.layout.activity_pwdstore) {
     private val currentDir: File
         get() = plist?.currentDir ?: getRepositoryDirectory(applicationContext)
 
-    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
-        if (resultCode == Activity.RESULT_OK) {
-            when (requestCode) {
-                // if we get here with a RESULT_OK then it's probably OK :)
-                BaseGitActivity.REQUEST_CLONE -> settings.edit { putBoolean(PreferenceKeys.REPOSITORY_INITIALIZED, true) }
-                // if went from decrypt->edit and user saved changes, we need to commitChange
-                REQUEST_CODE_DECRYPT_AND_VERIFY -> {
-                    if (data != null && data.getBooleanExtra("needCommit", false)) {
-                        if (data.getStringExtra("OPERATION") == "EDIT") {
```

Comment:
It seems to me that this `onActivityResult` logic has no correspondence anymore, is this intended?

Reply:
We don't have any other occurances of `"needCommit"` in the tree so this never gets hit.